pcsc-sys/src/lib.rs

```diff
@@ -41,7 +41,13 @@ pub type LONG = i32;
 #[cfg(target_os = "macos")]
 pub type ULONG = u32;
 
+#[cfg(target_os = "windows")]
+pub type SCARDCONTEXT = *const c_ulong;
+#[cfg(target_os = "windows")]
+pub type SCARDHANDLE = *const c_ulong;
+#[cfg(not(target_os = "windows"))]
```

Comment:
I think an empty line would be useful above this line, to delimit the two platforms (like in the `DWORD` etc. definitions).